packages/wonder-blocks-core/util/get-clickable-behavior.test.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import * as React from "react";
+import {MemoryRouter} from "react-router-dom";
+import ClickableBehavior from "../components/clickable-behavior.js";
+import getClickableBehavior from "./get-clickable-behavior";
+
+describe("getClickableBehavior", () => {
+    test("Without href, returns ClickableBehavior", () => {
+        // Arrange
+        const url = undefined;
+        const clientNav = undefined;
+        const router = undefined;
+        const expectation = ClickableBehavior;
+
+        // Act
+        const result = getClickableBehavior(url, clientNav, router);
+
+        // Assert
+        expect(result).toBe(expectation);
+    });
+
+    describe("with href", () => {
+        test("External URL, returns ClickableBehavior", () => {
+            // Arrange
+            const url = "http://google.com";
+            const clientNav = undefined;
+            const router = <MemoryRouter />;
+            const expectation = ClickableBehavior;
+
+            // Act
+            const result = getClickableBehavior(url, clientNav, router);
+
+            // Assert
+            expect(result).toBe(expectation);
+        });
+
+        describe("Internal URL", () => {
+            describe("Client navigation is undefined", () => {
+                test("No router, returns ClickableBehavior", () => {
+                    // Arrange
+                    const url = "/prep/lsat";
+                    const clientNav = undefined;
+                    const router = undefined;
+                    const expectation = ClickableBehavior;
+
+                    // Act
+                    const result = getClickableBehavior(url, clientNav, router);
+
+                    // Assert
+                    expect(result).toBe(expectation);
```

Comment:
I feel like `expect(result).toBe(ClickableBehavior)` is a clearer assertion b/c you don't have to look for the definition of `expectation` to know what it's asserting.

Reply:
Fair. I can make that change if you think it's important to do now. Otherwise, I'll take it and ensure that I do that in future work.

Reply:
I think it's probably worth making the change here too in case other people are looking at these tests as a model to follow.  I do like the organization of the tests into three separate parts.